bazel/envoy_build_system.bzl

```diff
@@ -43,7 +43,27 @@ def envoy_linkopts():
             # See note here: http://luajit.org/install.html
             "-pagezero_size 10000", "-image_base 100000000",
         ],
+        "//bazel:enable_exported_symbols": [
+            '-L/usr/local/lib/lua/5.1/',
```

Comment:
This won't work in many environments. Is there a more generic way to handle this?

Reply:
oops, Let me relook into this I was doing a lot of troubleshooting before and this just slipped by me :|